Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM resin/armhf-alpine:3.5
```

Comment:
I'd recommend trying armhf/alpine:3.5 which is an "official" version from the Docker team via @tianon